cluster-controller/src/test/java/io/strimzi/controller/cluster/operations/cluster/KafkaClusterOperationsTest.java

```diff
@@ -620,10 +623,7 @@ public void testReconcile(TestContext context) {
 
     private void reconcileCluster(TestContext context, ConfigMap originalCm, ConfigMap clusterCm) {
```

Comment:
Do we test reconcile only for Kafka clusters? Why?

Reply:
You are right. The other clusters reconcile tests were missing before this PR. Do we want to add them for this PR ?

Reply:
I think it would be good to add them in this PR if it is not too time consuming. But if nothing else we should at least raise an issue for it.